Frontend/ui-common/src/Components/RatingCard/View.purs

```diff
@@ -194,61 +178,50 @@ editTextView state push =
 
   ]
 
---------------------------------------------------- rideRatingButtonConfig ---------------------------------------------------
-
-rideRatingButtonConfig :: RatingCardState -> PrimaryButton.Config
-rideRatingButtonConfig state = let
-    config = PrimaryButton.config
-    primaryButtonConfig' = config
-      { textConfig
-        { text = (getString SUBMIT_FEEDBACK)
-        , accessibilityHint = "You Rated " <> show state.data.rating <> " stars : Submit Feedback : Button"
-        , color = if state.data.rating < 1 && state.data.appConfig.isGradient == "true" then "#696A6F" else state.data.appConfig.primaryTextColor
-        , width = MATCH_PARENT
-        }
-      , isClickable = if state.data.rating < 1 then false else true
-      , alpha = if not (state.data.rating < 1) || state.data.appConfig.isGradient == "true" then 1.0 else 0.4
-      , margin = (Margin 0 0 0 0)
-      , height = (V 48)
-      , gravity = CENTER
-      , isGradient = if state.data.rating < 1 then false else if state.data.appConfig.isGradient == "true" then true else false
-      , cornerRadius = state.data.appConfig.ratingConfig.buttonCornerRadius
-      , background = if state.data.rating < 1 && state.data.appConfig.isGradient == "true" then "#F1F1F4" else state.data.appConfig.primaryBackground
-      , id = "RideRatingButton"
-      , enableLoader = (getBtnLoader "RightRatingButton")
-      }
-  in primaryButtonConfig'
-
-------------------------
---------------------------
+------------------------starRatingView--------------------------
 
 starRatingView :: forall w . RatingCardState -> (Action  -> Effect Unit) -> PrestoDOM (Effect Unit) w
 starRatingView state push =
   linearLayout
     [ height WRAP_CONTENT
     , width MATCH_PARENT
     , orientation VERTICAL
-    , margin (MarginBottom 10)
     , gravity CENTER
     , padding (PaddingVertical 16 16)
     , cornerRadius 8.0
     ][ imageView [
-        imageWithFallback "ny_ic_driver_avatar,https://assets.juspay.in/beckn/nammayatri/user/images/ny_ic_driver_avatar.png"
+        imageWithFallback $ "ny_ic_driver_avatar,"<> (getAssetStoreLink FunctionCall)<>"ny_ic_driver_avatar.png"
         , height $ V 56
         , width $ V 56
         , cornerRadius 50.0
+        , visibility if state.showProfileImg then VISIBLE else GONE
       ]
-    ,textView $
-        [ height WRAP_CONTENT
-        , width $ V (screenWidth unit - 64)
-        , accessibilityHint $ "Rate Your Ride With " <> state.data.driverName
-        , accessibility ENABLE
-        , text $ getString RATE_YOUR_RIDE_WITH <> state.data.driverName
-        , color Color.black800
-        , maxLines 2
+    , linearLayout [
+        orientation HORIZONTAL
+      , width MATCH_PARENT
+      , height WRAP_CONTENT
+      , margin $ MarginBottom 10
+      , gravity CENTER
+      ][
+        textView $
+          [ height WRAP_CONTENT
+          , width $ V (screenWidth unit - 64)
+          , accessibilityHint state.title
```

Comment:
send `accessibilityHint` also from config. and only in English.

Reply:
Is this a common component for driver and customer now? accessibility is not required in provider app. Should this be a config of component here then?

Reply:
yes  it is common for customer and Driver.